megamek/src/megamek/common/Entity.java

```diff
@@ -10268,6 +10278,10 @@ public void addPilotingModifierForTerrain(PilotingRollData roll, Coords c,
         }
         IHex hex = game.getBoard().getHex(c);
         hex.terrainPilotingModifier(getMovementMode(), roll, enteringRubble);
+
+        if (hex.containsTerrain(Terrains.JUNGLE) && getCrew().getOptions().booleanOption(OptionsConstants.PILOT_TM_FOREST_RANGER)) {
+            roll.addModifier(-1, "Forest Ranger");
```

Comment:
The tooltip for forest ranger didn't really elude to this, bit the tooltip for Mountaineer did.  Should mountaineer also have a check for rubble here?  Perhaps I don't understand what this modifier is for.

Reply:
Jungle has a PSR modifier unlike woods. Forest Ranger gives a bonus to that roll. If its not in the tooltip I will add it. Pretty sure I had it there at one point though...

Reply:
This was corrected with 01c807d